Resources/public/ts/react/GridComponent.tsx

```diff
@@ -63,6 +64,13 @@ export class GridComponent extends React.Component<IProps, IState> {
     }
 
     componentDidMount() {
+        if (this.props.data) {
```

Comment:
That change can be pretty dangerous and I don't see a case where you have to overwrite the configuration from the JS side. Especially because this fetchColumnDefs call in the last line of this method, will may (because of timing) revert any changes made before.

Reply:
This should be removed

Reply:
I'm sorry. This shouldn't appear. I'll fix it...